### PR TITLE
Fetch project with active team leader

### PR DESCRIPTION
**Pull request by juliepromact**
Fixes#https://github.com/Promact/slack-erp-custom-integration-mvc/issues/138.

**Files Changed :**

**1. ProjectRepository.cs** - Condition added to fetch project with active team leader.
**2. ProjectTests.cs** - Updated test cases.

### CODE REVIEW

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs [hunk 1]
```diff
@@ -63,8 +63,8 @@ public class ProjectRepository : IProjectRepository
                     userAc.LastName = _stringConstant.TeamLeaderNotAssign;
                     userAc.Email = _stringConstant.TeamLeaderNotAssign;
                 }
-                var CreatedBy =(await _userDataRepository.FirstOrDefaultAsync(x => x.Id == project.CreatedBy))?.FirstName;
-                var UpdatedBy =(await _userDataRepository.FirstOrDefaultAsync(x => x.Id == project.UpdatedBy))?.FirstName;
+                var CreatedBy = (await _userDataRepository.FirstOrDefaultAsync(x => x.Id == project.CreatedBy))?.FirstName;
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: wrong naming conventions and variable is not used anywhere

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs [hunk 2]
```diff
@@ -63,8 +63,8 @@ public class ProjectRepository : IProjectRepository
...
+                var UpdatedBy = (await _userDataRepository.FirstOrDefaultAsync(x => x.Id == project.UpdatedBy))?.FirstName;
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: same as above

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs [hunk 3]
```diff
@@ -63,8 +63,8 @@ public class ProjectRepository : IProjectRepository
...
                 DateTime? UpdatedDate;
                 if (project.UpdatedDateTime == null)
```
Comment (chintans) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: This entire if and else block is useless

File: Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs (hunk 1 above)
Comment (juliepromact) [inline Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/ProjectsRepository/ProjectRepository.cs]: Rebase issue. Solved.

**Pull request merged**